apps/web/src/components/Feed/Socialize/CommentsModal/PostCommentsModal.tsx

```diff
@@ -57,6 +58,8 @@ export default function PostCommentsModal({
     queryRef
   );
 
+  useClearURLQueryParams('commentId');
```

Comment:
this ensures the query param gets cleared out once the `PostCommentsModal` opens. otherwise, the query will continue to linger in the URL and always open the comments modal whenever the user tries to post